pages/components/pill.html

```diff
@@ -0,0 +1,95 @@
+<html lang="en">
+  <%- include('head.html'); -%>
+  <body>
+    <script type="module" src="../sidebar-element.js"></script>
+    <script>
+        document.querySelector('html').addEventListener('w-pill-click', e => {
+            console.log('Click clicked');
+        });
+        document.querySelector('html').addEventListener('w-pill-close', e => {
+            console.log('Close clicked');
+        });
```

Comment:
So if we have multiple pill components, the same message will be logged in the console, regardless which pill is closed. What if we wanted to log a different message for one of the pills? Would it work by passing a `@w-pill-close="console.log('close clicked pill 1')"` on one of them?

Reply:
Ideally everyone would use these events like `<w-pill @w-pill-click="doSomething()">` which is the lit way of doing this. However for the sake of our documentation site, I didn't want to bring in lit just to use these handlers, hence doing it plain js way. 

To define multiple elements specific event handlers, you can tie them to the dom element instead of the whole document :)